README.md

```diff
@@ -1 +1,12 @@
-# interactors
\ No newline at end of file
+# Interactors
+
+[![CircleCI](https://circleci.com/gh/thefrontside/interactors.svg?style=shield)](https://circleci.com/gh/thefrontside/interactors)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Created by Frontside](https://img.shields.io/badge/created%20by-frontside-26abe8.svg)](https://frontside.com)
+[![Chat on Discord](https://img.shields.io/discord/700803887132704931?Label=Discord)](https://discord.gg/Ug5nWH8)
+
+Interactors is a library designed to help you write UI tests from the outside in that are both efficient and valuable.
+
+Interactors manipulate an HTML interface from the perspective of a user and make no assumptions about the internal workings of an app. This means they cover 100% of your UI code starting from the raw input event handlers. Your tests will work the same no matter if your application is written in React, Angular, Vue, or whether you chose to eventually rewrite it in Framework 2043.™
```

Comment:
```suggestion
Interactors manipulate an HTML interface from the perspective of a user and makes no assumptions about the internal workings of an app. This means they cover 100% of your UI code starting from the raw input event handlers. Your tests will work the same no matter if your application is written in React, Angular, Vue, or whether you choose to eventually rewrite it in Framework 2043.™
```

👆? I'm not sure

Reply:
New commit pushed to stick to "Interactors" as plural. Reads more naturally.